config/routes.rb

```diff
@@ -11,7 +11,7 @@
 
       get 'admin', to: 'admin#index'
```

Comment:
Shouldn't this also be changed?  to `...   to: membership_applications#index`

I'm not sure that we can delete this because there are routes with `/admin/....` in them. (Since you deleted this action from the controller, my first inclination was to delete this route, too.)  I haven't thought through the effect of deleting it; there may not be any ill effects.

Reply:
There are no ill effects that I (or the tests) can think of.